carmm/run/workflows/react.py

```diff
@@ -287,6 +288,8 @@ def get_mulliken_charges(self, atoms: Atoms, restart: bool = True):
 
         counter, out, subdirectory_name, initial = helper.restart_setup()
 
+        print(out)
```

Comment:
Presumably leftover from the debugging process

Reply:
Removed.